pkgs/development/interpreters/python/wrapper.nix

```diff
@@ -23,17 +23,29 @@ let
       fi
       mkdir -p "$out/bin"
 
-      for path in ${stdenv.lib.concatStringsSep " " paths}; do
-        if [ -d "$path/bin" ]; then
-          cd "$path/bin"
-          for prg in *; do
-            if [ -f "$prg" ]; then
-              rm -f "$out/bin/$prg"
-              makeWrapper "$path/bin/$prg" "$out/bin/$prg" --set PYTHONHOME "$out"
-            fi
+      # Generate new wrapper scripts for the all the executables found in our
```

Comment:
I don't understand why this modification is a part of this PR. Is this change *necessary* after the modification of `makeWrapper`? Does it depend on the change of `makeWrapper`? If either of these answers is "no", then this modification should be a separate PR.